monitoring-plugins/check_openshift.py

```diff
@@ -67,10 +69,26 @@
                     help='Check status of all nodes')
 PARSER.add_argument("--check_pods", action='store_true',
                     help='Check status of pods ose-haproxy-router and ose-docker-registry')
+PARSER.add_argument("--exclude_pods", type=str,
+                    help='Exclude pods where the name contains the string passed in argument, find in the field deploymentconfig labels str|str|str ')
```

Comment:
It would be nice to have the capability to provide a label/selector like in 
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
For example:
```
-l 'environment in (production),tier in (frontend)'
```